src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile.cs

```diff
@@ -339,6 +339,25 @@ public async Task When_Open_By_Encoded_URI_With_Space()
 			}
 		}
 
+		[TestMethod]
+		public async Task When_Open_Multiple_Reads_Single_Write()
+		{
+			var uri = new Uri($"ms-appx:///Assets/Asset With Spaces.svg");
+
+			try
+			{
+				var file = await StorageFile.GetFileFromApplicationUriAsync(uri);
+				_ = await file.OpenStreamForReadAsync();
+				_ = await file.OpenStreamForReadAsync();
+				_ = await file.OpenStreamForWriteAsync();
+				_ = await file.OpenStreamForReadAsync();
+			}
+			catch (Exception ex)
+			{
+				Assert.Fail(ex.Message);
```

Comment:
BTW you don't need to do that: exception are catch by the test engine and are automatically reported as failure.